src/tests/ftest/util/server_utils.py

```diff
@@ -157,6 +158,20 @@ def using_dcpm(self):
             value = self.yaml.using_dcpm
         return value
 
+    def get_interface_envs(self, index=0):
+        """Get the environment variable names and values for the interfaces.
+
+        Args:
+            index (int, optional): server index from which to obtain the
+                environment variable values. Defaults to 0.
+
+        Returns:
+            EnvironmentVariables: a dictionary of environment variable names
+                and their values extracted from the daos_server yaml
+                configuration file.
+
+        """
+        return self.yaml.get_interface_envs(index)
```

Comment:
Why is this being added?  It looks like it came from an outdated copy of the file.